lib/templates/facebook/koala.rb

```diff
@@ -0,0 +1,8 @@
+Koala.configure do |config|
+  config.access_token = 'MY_TOKEN'
+  config.app_access_token = 'MY_APP_ACCESS_TOKEN'
+  config.app_id = 'MY_APP_ID'
+  config.app_secret = 'MY_APP_SECRET'
```

Comment:
I'd suggest using the `ENV` here, and maybe we could add the keys to the `application.yml.example`?

Reply:
Fixed, and as we talked keys are already in application.yml.example on rails_api_base repo